src/utils/summarizer.py

```
@@ -66,7 +66,7 @@ class SummaryType(Enum):
     model=settings.SUMMARY.MODEL,
     max_tokens=settings.SUMMARY.MAX_TOKENS_SHORT,
 )
-async def create_short_summary(
+async def create_short_summary(  # pragma: no cover
```

Ruff (0.12.2)

Missing return type annotation for private function `create_short_summary`

Add return type annotation: `str`

(ANN202)